index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Website Design Project</title>
+        <link rel="stylesheet" href="style.css" type="text/css">
+    </head>
+    <body>
+        <header class="imagecontainer">
+            <section class="img defineimage"></section>
+            <section class="img defineimage1"></section>
+        </header>
+        <section class="textcontainer">
+            <h1 class="text text1">Rajendra Medini</h1>
+            <h1 class="text text3">Projects</h1>
+            <p class="text text2">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
```

Comment:
```suggestion
            <h1>Rajendra Medini</h1>
            <h1>Projects</h1>
            <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my works. If you like what you see and have a project you need to be coded, don’t hesitate to contact me.</p>
```
text as a class is not styled in the CSS file.
Also, text1, text2, text3 all have the same CSS styling which you can actually choose to leave out. Your work will still look good. That is because, for these kinds of attributes, you do not need to focus on their positioning. You have two h1 tags and a paragraph. 
Please make these few edits.

Reply:
I have removed the text from the HTML class tag as per your advice. But, I cannot remove text1, text2, text3 because removing it collapses my styling and they lose their positioning. It becomes clumsy if I remove them, so I would like to keep them as it is, since you have suggested that I can choose to leave out.